README.md

```diff
@@ -80,6 +79,7 @@ For now (start of april 2021) we can use HellSpawner to:
     *   DAT - palettes
     *   PL2 - palette transforms
     *   DT1 - map tiles
+    *   DS1 - map data
```

Comment:
Map "data" is vague.

A DS1 is a map "preset", or "stamp," which describes a static configuration of tiles from one or many external DT1 files.

Reply:
https://github.com/OpenDiablo2/HellSpawner/pull/320/commits/0696184b5deb84a96c95ffabc7bbfec6c493b2ce